### DISCUSSION TITLE
NextJS project, Pages/Api the folders are uploaded empty

### DISCUSSION THREAD

**Discussion by JavierPalina** [vercel/community#1670] [Help]
### Summary

I can't understand why in next 13 the pages/api folders are uploaded completely empty, can someone help me with this?
├ λ /api/blogs                                      0 B            79.2 kB
--
01:45:43.674 | ├ λ /api/blogs/[id]                                 0 B            79.2 kB
01:45:43.674 | ├ λ /api/blogscategories                            0 B            79.2 kB
01:45:43.674 | ├ λ /api/blogscategories/[id]                       0 B            79.2 kB
01:45:43.675 | ├ λ /api/categories                                 0 B            79.2 kB
01:45:43.676 | ├ λ /api/categories/[id]                            0 B            79.2 kB
01:45:43.676 | ├ λ /api/chatbot                                    0 B            79.2 kB
01:45:43.676 | ├ λ /api/chatbot/[id]                               0 B            79.2 kB
01:45:43.676 | ├ λ /api/contenido                                  0 B            79.2 kB
01:45:43.676 | ├ λ /api/contenido/[id]                             0 B            79.2 kB
01:45:43.676 | ├ λ /api/products                                   0 B            79.2 kB
01:45:43.676 | ├ λ /api/products/[id]                              0 B            79.2 kB

**Comment by amyegan** [comment]
Hi Javier. Are you experiencing a problem, or simply wondering why the size is 0 B? If this is related to an error or deployment problem, please share a minimal reproduction to help me investigate 🙏

**Reply by JavierPalina** [reply]
Hi Amy, I am doing a crud in Nextjs, with mongoose, mongodb and nodejs, on the development server it works perfectly, but when I do the deployment to vercel, the functionality buttons stop working, I was seeing that it is because the files of the Api are uploading empty (they only start working when I go to the route https:mypagevercel.com/api/products) after going to that route, for some reason the products start working... but the admin I'm using to upload products still not working
